mars/deploy/oscar/tests/test_local.py

```diff
@@ -421,23 +424,26 @@ def test_sync_execute():
         assert d is c
         assert abs(session.fetch(d) - raw.sum()) < 0.001
 
-        with tempfile.TemporaryDirectory() as tempdir:
-            file_path = os.path.join(tempdir, "test.csv")
-            pdf = pd.DataFrame(
-                np.random.RandomState(0).rand(100, 10),
-                columns=[f"col{i}" for i in range(10)],
-            )
-            pdf.to_csv(file_path, index=False)
-
-            df = md.read_csv(file_path, chunk_bytes=os.stat(file_path).st_size / 5)
-            result = df.sum(axis=1).execute().fetch()
-            expected = pd.read_csv(file_path).sum(axis=1)
-            pd.testing.assert_series_equal(result, expected)
-
-            df = md.read_csv(file_path, chunk_bytes=os.stat(file_path).st_size / 5)
-            result = df.head(10).execute().fetch()
-            expected = pd.read_csv(file_path).head(10)
-            pd.testing.assert_frame_equal(result, expected)
+        if config["incremental_index"]:
+            with tempfile.TemporaryDirectory() as tempdir:
+                file_path = os.path.join(tempdir, "test.csv")
+                pdf = pd.DataFrame(
+                    np.random.RandomState(0).rand(100, 10),
+                    columns=[f"col{i}" for i in range(10)],
+                )
+                pdf.to_csv(file_path, index=False)
+
+                df = md.read_csv(file_path, chunk_bytes=os.stat(file_path).st_size / 5)
+                result = df.sum(axis=1).execute().fetch()
+                expected = pd.read_csv(file_path).sum(axis=1)
+                pd.testing.assert_series_equal(result, expected)
+
+                df = md.read_csv(file_path, chunk_bytes=os.stat(file_path).st_size / 5)
+                result = df.head(10).execute().fetch()
+                expected = pd.read_csv(file_path).head(10)
+                pd.testing.assert_frame_equal(result, expected)
+        else:
+            print("Incremental index is not supported.")
```

Comment:
Is this print intended or just for test?

Reply:
I think it's better to print this if the incremental index is not supported / tested. I am working on the Ray backend to support the incremental index feature as soon as possible. There is a TODO comment in the Ray test case.

Reply:
How about add a TODO instead of printing sth?

Reply:
There is  already a TODO comment in the Ray test case: https://github.com/mars-project/mars/pull/2976/files#diff-c1dddc59766fc396dab0a8b1c8f3ffe0a9256c72a07feb11d76b0239fff5a437R106

I will remove this print as you have concerns.

Reply:
The print has been removed.